server/controller/module/sendMail.ts

```diff
@@ -0,0 +1,59 @@
+import { createTransport } from "nodemailer";
+import * as dotenv from "dotenv";
+import path from "path";
+
+//The below config setup was unable to be resolved. Need to connect and see on this
+
+const config = dotenv.config({
+  path: path.resolve(__dirname, "./env"),
+}).parsed;
+
+export default async function sendMail(reci: String[]) {
+  var transport = {
+    host: config?.SMTP_HOST,
+    secureConnection: false,
+    port: config?.SMTP_PORT,
+    tls: {
+      ciphers: "SSLv3",
+    },
+    auth: {
+      user: config?.MAILER_USER,
+      pass: config?.MAILER_PASS,
+    },
+  };
+  var transporter = createTransport(transport);
+
+  var toObj = "";
```

Comment:
Why do we need this? we can simply have a list of email ids to which we need to send mail to that will be an argument to the middleware service maybe or we can explicitly send it inside request object?